### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by betses**
# Features Added on this branch 

- setup eslint environment
- Installed and configured Webpack
- Add todolist from object 
- Add styling to webpack and edit
- edited git ignore asdSS

Kindly please review my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```
Comment (zieeco) [inline README.md]: - Please, kindly remove this line as it's an instruction for you to use to modify your readme

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+   Clone the website in yoiur repost
+
+### Install
+  rum np
```
Comment (zieeco) [inline README.md]: - Please, kindly fix this typo. I supposed you meant `run npm install`

File: README.md [hunk 3]
```diff
@@ -1 +1,70 @@
...
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```
Comment (zieeco) [inline README.md]: - Please, kindly remove these headers as they're not being used yet.

File: README.md [hunk 4]
```diff
@@ -1 +1,70 @@
...
+
+## Authors
+
```
Comment (zieeco) [inline README.md]: - kindly make the authors singular as you've worked on the project alone

File: README.md [hunk 5]
```diff
@@ -1 +1,70 @@
...
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/betses)
+- Twitter: [@twitterhandle](https://twitter.com/betse_s)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (zieeco) [inline README.md]: - Please, kindly update this issue page to to refer to the To Do list repo's issues page.

File: README.md [hunk 6]
```diff
@@ -1 +1,70 @@
...
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (zieeco) [inline README.md]: - Please, kindly follow the instruction to add a licensed content as the link is currently broken now.

File: README.md [hunk 7]
```diff
@@ -1 +1,70 @@
...
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+   Clone the website in yoiur repost
+
```
Comment (zieeco) [inline README.md]: - Please, kindly fix these typos and make your sentence meaningful.

File: README.md [hunk 8]
```diff
@@ -1 +1,70 @@
...
+> A todo webapp configured with webpack
+
+
+## Built With
+
+- Major languages
+  -Java Script 
+HTML 
+- Frameworks
+- Technologies used
```
Comment (zieeco) [inline README.md]: - Please, kindly remove the "Framework" as you've not used one. You can also remove the Technologies or simply add the technologies you've used.

File: README.md [hunk 9]
```diff
@@ -1 +1,70 @@
...
+
+# Project Name
+ToDo App
+> A todo webapp configured with webpack
+
+
+## Built With
+
+- Major languages
+  -Java Script
```
Comment (zieeco) [inline README.md]: - Please, kindly correct the spelling of "JavaScript" as supposed to "Java Script"

File: src/index.js [hunk 10]
```diff
@@ -0,0 +1,68 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (zieeco) [inline src/index.js]: - Please, let's try to write codes that conform to the eslint as it helps us write readable and maintainable code. Please, remove and fix the linter issue

File: src/index.js [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+const loadTodo = (parent) => {
+  todo.sort((a, b) => a.index - b.index);
+
+  todo.forEach((task) => {
+    const el = document.createElement('li');
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.id = `task-${task.index}`;
+    // eslint-disable-next-line no-use-before-define
```
Comment (zieeco) [inline src/index.js]: - Please, let's try to write codes that conform to the eslint as it helps us write readable and maintainable code. Please, remove and fix the linter issue

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @<STUDENT NAME>,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @betses.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- You have implemented all suggestions from the previous reviewer 💯 🥇 

### Optional suggestions

- Please, consider fixing the broken link to the MIT.md file in your README file 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**